src/docs/src/ddocs/mango.rst

```diff
@@ -0,0 +1,922 @@
+``"partial_filter_selector"`` field:
+
+.. code-block:: http
+
+        POST /db/_index HTTP/1.1
+        Content-Type: application/json
+        Content-Length: 144
+        Host: localhost:5984
+
+        {
+          "index": {
+            "partial_filter_selector": {
+              "status": {
+                "$ne": "archived"
+              }
+            },
+            "fields": ["type"]
+          },
+          "ddoc" : "type-not-archived",
+          "type" : "json"
+        }
+
+Partial indexes are not currently used by the query planner unless specified
+by a ``"use_index"`` field, so we need to modify the original query:
+
+.. code-block:: javascript
+
+    {
+        "selector": {
+            "status": {
+                "$ne": "archived"
+            },
+            "type": "user"
+        },
+        "use_index": "type-not-archived"
+    }
+
+Technically, we do not need to include the filter on the ``"status"`` field
+in the query selector - the partial index ensures this is always true -
+but including it makes the intent of the selector clearer and will make
+it easier to take advantage of future improvements to query planning
+(e.g. automatic selection of partial indexes).
+
+.. note::
+    An index with fields is only used, when the selector includes
+    all of the fields indexed. For instance, if an index contains ``["a", "b"]``
+    but the selector only requires field ``["a"]`` to exist in the matching
+    documents, the index would not be valid for the query. All indexes,
+    however, can be treated as if they include the special fields ``_id`` and
+    ``_rev``. They **never** need to be specified in the query selector.
+
+.. _ddoc/mango/indexes/nouveau:
+
+Nouveau Indexes
+---------------
+
+Mango can also interact with the :ref:`Nouveau search system <ddoc/nouveau>`,
+using the :ref:`$text selector <find/text>` and Nouveau indexes. These indexes
+can be queried using either ``$text`` or
+:http:get:`/{db}/_design/{ddoc}/_nouveau/{index}`.
+
+Example index:
+
+.. code-block:: javascript
+
+    {
+        "type": "nouveau",
+        "index": {
+            "fields": [
+                {"name": "foo", "type": "string"},
+                {"name": "bar", "type": "number"},
+                {"name": "baz", "type": "string"},
+            ],
+            "default_analyzer": "keyword",
+        }
+    }
+
+A Nouveau index definition consists of:
+
+* **fields**: ``"all_fields"`` or list of objects:
+
+  * **name** (`string`): not blank
+  * **type** (`string`): one of ``"text"``, ``"string"``, ``"number"``, ``"boolean"``
+
+* **default_analyzer** (`string`): Nouveau analyzer to use, defaults to ``"keyword"``
+* **default_field**: boolean or object of ``enabled`` and ``analyzer``
+* **partial_filter_selector** (`object`): A :ref:`selector<find/selectors>`, causing this
+  to be a :ref:`partial index<find/partial_indexes>`
+* **selector** (`object`): A :ref:`selector<find/selectors>`
```

Comment:
This description is the same for text type indexes, so we could duplicate it as another section. [Cloudant's documentation](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-query#creating-a-type-text-index) has a fuller description around the default values and how Mango translates fields/queries into Lucene.

Reply:
Ok, I'll update the language to refer to both.